lib/components/Editor/CustomExtensions/Variable/VariableList.jsx

```diff
@@ -34,15 +35,15 @@ const VariableCategory = ({ index, title, variables, onClickItem }) => (
     {title && <h6>{title}</h6>}
     <div className="neeto-editor-variables-row">
       {variables.map(item => (
-        <button
-          className="neeto-editor-variable"
+        <Tag
           data-cy={`neeto-editor-variable-option-item-${item.label}--${item.value}`}
           key={`${item.label}--${item.value}`}
-          type="button"
+          label={item.label}
+          style="secondary"
+          className="neeto-ui-text-black
+                    neeto-ui-border-gray-300 hover:neeto-ui-text-primary-800 hover:neeto-ui-border-primary-800 neeto-ui-leading-snug cursor-pointer px-2 ne_variable-tag"
```

Comment:
@praveen-murali-ind _a please review.

If I understand correctly, we need not override any of the default styles of the `Tag` component and let neetoUI handle the styling.

@gaagul please delete the class definition of unused styles (if any). In this case the `neeto-editor-variable` class is deleted.